jvb/src/main/kotlin/org/jitsi/videobridge/cc/allocation/Allocation.kt

```diff
@@ -39,8 +39,7 @@ class Allocation(
             allocations.all { allocation ->
                 other.allocations.any { otherAllocation ->
                     allocation.endpointId == otherAllocation.endpointId &&
-                        allocation.targetLayer?.index == otherAllocation.targetLayer?.index &&
-                        allocation.idealLayer?.index == otherAllocation.idealLayer?.index
+                        allocation.targetLayer?.index == otherAllocation.targetLayer?.index
```

Comment:
Just double-checking -- a change in ideal layers can change whether we want to send bwe probing.  Is this code separate from that?

Reply:
Yes, the probing samples the ideal layers via the current allocation. 

There is a weird situation with probing, that I briefly mention in `BitrateController.kt`. The ideal layers are updated with the allocation, and inactive layers are not considered. So when a higher layer becomes active, we will not probe for it until the allocation updates. This shouldn't take very long because we `maybeUpdate()` on each SR (no more often than once every 15 seconds). Conversely, if the ideal layer becomes inactive, we will probe for 0 bps for that endpoint until the allocation updates. I think this is an improvement over the previous behavior, which was to consider inactive layers for allocation, leading to probing for 0 indefinitely as long as the layer is inactive (but pick-up the bitrate immediately when it becomes active).

Reply:
I'll add the above description somewhere in the code